### ISSUE TITLE
network sets tabindex=900 which harms accessibility

### ISSUE THREAD

**Issue by krabina** [ProfessionalWiki/Network#14]
the network visualzation sets further style elements, including **tabindex="900"**

`<div class="vis-network" style="position: relative; overflow: hidden; touch-action: pan-y; user-select: none; width: 100%; height: 300px;" tabindex="900"><canvas style...</canvas></div>`

Due to an accessiblilty expert, this is a no-go. If you want to have "vis-network" in tabindex, then it should be tabindex="0" (not greater than 0)

Interestingly, I cannot find and tabindex setting in the network source code (and no indication in vis.js. So I wonder where this even comes from...

**Comment by JeroenDeDauw** [comment]
Is coming from the network JS library. https://raw.githubusercontent.com/ProfessionalWiki/Network/master/resources/lib/vis-network.js Search "tabIndex"

```js
this.frame.tabIndex = 900; // tab index is required for keycharm to bind keystrokes to the div instead of the window
```

Could just remove the line or change the value to 0 in the copy of the library we have. Presumably breaks whatever this line was added to fix. 

Probably best if you take this issue upstream https://github.com/visjs/vis-network

**Comment by krabina** [comment]
was resolved in upstream release 8.5.5: https://github.com/visjs/vis-network/issues/1238
